frontend/airtable/utils/index.js

```diff
@@ -61,3 +70,61 @@ export const findTeam = async (teamId) => {
     console.error(error);
   }
 };
+
+export const findUserByPhoneNumber = async phoneNumber => {
+  try {
+    const { data } = await client.query({
+      query: SEARCH_PHONE_NUMBER,
+      variables: {
+        phoneNumber,
+      },
+    });
+    return data;
+  } catch (error) {
+    console.error(error);
+  }
+};
+
+export const createDummyRecord = async (phoneNumber, teamId) => {
```

Comment:
Instead of dummy, the right term here would be anonymous. Dummy implies that this is sort of like mock data used for development. Anonymous is used when you have a user with no known identity. For example if you were buying something on an e-commerce site and you have added items to your cart but you have not yet logged in, you would be considered an anonymous user.